src/sha1_hash.cpp

```diff
@@ -75,14 +75,12 @@ namespace libtorrent
 				continue;
 			}
 
-#if TORRENT_HAS_SSE
-#ifdef __GNUC__
+#if TORRENT_HAS_BUILTIN_CLZ
 			return ret + __builtin_clz(v);
-#else
+#elif TORRENT_HAS_SSE && defined _MSC_VER
```

Comment:
I got the impression from your last commit that this isn't an SSE feature, so maybe it shouldn't be tested here